cfme/tests/containers/test_reload_button_provider.py

```diff
@@ -41,10 +41,8 @@ def test_reload_button_provider(provider):
     list_img_from_openshift_parsed = [i[7:].split(
         '@sha256:')[-1] for i in list_img_from_openshift_splitted]
 
-    for s in list_img_from_openshift_parsed:
-        for item in list_img_from_registry_splitted:
-            if item not in list_img_from_openshift_parsed:
-                list_img_from_openshift_parsed.append(item)
+    list_img_from_openshift_parsed.extend([item for item in list_img_from_registry_splitted
```

Comment:
this transformation is not equivalent
the nested for loop ensures global uniqueness,
while the list expression allows duplicate list elements

the outermost loop however seems to have been misplaced to begin with
since the test however does not seem to work with the elements in order,
i propose using a set here, and just updating it